zproject/test_settings.py

```diff
@@ -18,6 +18,8 @@
     os.environ["EXTERNAL_HOST"] = "testserver"
 from .settings import *
 
+FAKE_EMAIL_DOMAIN = "zulip.testserver"
```

Comment:
If Casper is failing, it's possible this needs to be inside the Casper conditional block.

Reply:
Indeed, turned out I just had to change some hard-coded bot emails in casper tests to this domain.